batchupload/make_info.py

```diff
@@ -69,7 +69,7 @@ def load_data(self, in_file):
         pass
 
     @abstractmethod
-    def load_mappings(self, update=True):
+    def load_mappings(self, update):
```

Comment:
We are. But rather than having various defaults all over the place this is now uniquely defined in [L241](https://github.com/lokal-profil/BatchUploadTools/pull/23/files/115df090e1ddb11444e225d2ccf373ce27519efa#diff-da8edd19474bff18a136d1d3f606f054R241)